### PR TITLE
Declarative API for specifying nested fields

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

**Comment by taion** [comment]
cc @cancan101

File: flask_jsonapiview/view.py [hunk 1]
```diff
@@ -108,6 +108,8 @@ class ModelView(ApiView):
     model = None
     url_id_key = 'id'
 
+    nested = None
```
Comment (cancan101) [inline flask_jsonapiview/view.py]: Make this `{}`.

Comment (cancan101) [inline flask_jsonapiview/view.py]: or frozen dict.

Comment (taion) [inline flask_jsonapiview/view.py]: That's mutable though...

Comment (cancan101) [inline flask_jsonapiview/view.py]: http://werkzeug.pocoo.org/docs/0.10/datastructures/#werkzeug.datastructures.ImmutableDict

Comment (taion) [inline flask_jsonapiview/view.py]: Think it's worth keeping the check at https://github.com/4Catalyzer/flask-jsonapiview/pull/26/files#diff-7e2c0e4609fe4eecb154d2d2534b07d4R183, then?

**Pull request merged**